src/participant_network.star

```diff
@@ -207,6 +207,11 @@ def launch_participant_network(
             participant.el_extra_env_vars,
         )
 
+        # Add participant el additional prometheus metrics
```

Comment:
i see this is a smaller change than changing every client to add the prometheus config properly - is that why we are doing this?

Reply:
Yes, it just passing the prometheus config from a participant to its contexts node metrics info. Since it was a simple association adding that as a param means that every client would have to do this step which is essentially the same. If there is a better place for it I can move it.

Reply:
No I am fine keeping it here! Just a bit afraid of people adding more and more things here; but we can address that fear later.